src/main/java/ru/practicum/shareit/item/service/ItemServiceImpl.java

```diff
@@ -0,0 +1,97 @@
+package ru.practicum.shareit.item.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ru.practicum.shareit.exceptions.NoSuchItem;
+import ru.practicum.shareit.exceptions.NoSuchUser;
+import ru.practicum.shareit.exceptions.WrongUser;
+import ru.practicum.shareit.item.dto.ItemDto;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.item.model.ItemMapper;
+import ru.practicum.shareit.user.service.UserService;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class ItemServiceImpl implements ItemService {
+    private final Map<Long, Item> items = new HashMap<>();
+    private final ItemMapper itemMapper;
+    private final UserService userService;
+    public long idCounter = 1;
+
+    public Item createItem(long userId, ItemDto itemDto) {
+        if (!userService.checkUserExists(userId)) {
+            log.error("User {} was not found", userId);
+            throw new NoSuchUser("No such user");
+        }
+        Item item = itemMapper.toItem(itemDto);
```

Comment:
Для использования статических методов не нужно создавать экземпляр класса. Можно вызывать из через method reference:
```suggestion
        Item item = ItemMapper.toItem(itemDto);
```